tools/appveyor.psm1

```diff
@@ -517,16 +517,23 @@ function Push-Artifact
     param(
         [Parameter(Mandatory)]
         [ValidateScript({Test-Path -Path $_})]
-        $Path
+        $Path,
```

Comment:
I think this would be more clear if `$Path` was named `$ArtifactPath` and `$Name` to `$ArtifactName`.